DiscordIan/Model/OpenWeatherMap/OpenWeatherMapForecastModel.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
+    using System.Collections.Generic;
+
+    using System.Globalization;
+    using Newtonsoft.Json;
```

Comment:
We probably do not need to bring Newtonsoft in, .NET Core 3.1 has [System.Text.Json](https://docs.microsoft.com/en-us/dotnet/api/system.text.json?view=netcore-3.1).